packages/nimble-components/src/anchor/template.ts

```diff
@@ -40,6 +43,8 @@ AnchorOptions
         aria-owns="${x => x.ariaOwns}"
         aria-relevant="${x => x.ariaRelevant}"
         aria-roledescription="${x => x.ariaRoledescription}"
+        @mouseenter="${x => x.updateContentEditable()}"
+        @focus="${x => x.updateContentEditable()}"
```

Comment:
I dunno, this implementation feels like a whack-a-mole kind of edge case. An immediate concern is missing symmetric event handlers, mouseenter / mouseleave, focus/blur. But I don't have confidence that covers all interactions anyway.

I guess we should be considering how contenteditable applies to all of our components and this doesn't seem like a pattern that is easy to scale. At the same time this is really an edge case as far as I'm aware. I don't think 99% of usage of anchors cares about this but they all will have event listeners set-up to track this.

Can we just make this explicit instead? I'm pretty sure the rich-text-editor can set attributes it wants on the element it emits. Can it just emit the contenteditable attribute and we reflect it internally? Just make it a simple attribute reflection?

We can create usage notes saying due to browser limitations you have to explicitly set contenteditable on the nimble-anchor.

Reply:
I like this very much. 👍